### ISSUE TITLE
Input underline

### ISSUE THREAD

**Issue by justfede** [GeekyAnts/NativeBase#611]
Hey guys, I want to change the underline color for my "Input" component but i can't.

Here's my code:

    <Item floatingLabel>
    <Label>username</Label>
    <Input underlineColorAndroid="#00ffff" onChangeText={(text) => this.setState({username:text})} />
    </Item>`

But the property underlineColorAndroid don't make anything (border bottom still gray):

<img width="297" alt="underline" src="https://cloud.githubusercontent.com/assets/400900/23686010/e711200c-0385-11e7-930e-9be85b68e776.png">

Anyone knows how to change it?

Thanks!

**Comment by sankhadeeproy007** [comment]
`<Item floatingLabel style={{borderColor: '#00fff'}}>`
Should work for you.

**Comment by HyperSimon** [comment]
@sankhadeeproy007 Can you tell me how to make the underline of input disappear? 

I tried this code but not work
```js
          <Item underline={false}}>
            <Icon active
                  style={{marginLeft: 10, color: '#8E8E93'}}
                  name='search'/>
            <Input placeholder={this.getString('name')}/>
          </Item>
```

and this is my code now, 
```js
          <Item style={{borderWidth:0}}>
            <Icon active
                  style={{marginLeft: 10, color: '#8E8E93'}}
                  name='search'/>
            <Input placeholder={this.getString('name')}/>
          </Item>
```

Is there any NativeBase way to solve this problem？

**Comment by amanthegreatone** [comment]
this does it for me `<Item style={{ backgroundColor: 'grey', borderColor: 'transparent' }}>`

**Comment by amorenew** [comment]
also this work:
                        backgroundColor: 'transparent',
                        borderColor: 'rgba(255,255,255,.4)'

**Comment by avdeveloper** [comment]
I think the comments above are bandaids to a problem that could be solved at the cause. Why are the props `underlineColorAndroid` and `placeholderTextColor` placed below `{...this.props}` in here? https://github.com/GeekyAnts/NativeBase/blob/master/src/basic/Input.js#L22

**Comment by SupriyaKalghatgi** [comment]
Fixed with NB [2.3.10](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.10)

**Comment by ramanasak** [comment]
<Item style={{ borderColor: 'transparent'}}> this should work
